content/community/news/science-updates/interchange-update-2021-11-10/index.md

```diff
@@ -258,53 +256,57 @@ from openff.interchange.drivers.lammps import get_lammps_energies
 ```python
 print(get_gromacs_energies(interchange))
 ```
+
     Energies:
 
-    Bond:          		17.31004524230957 kJ / mol
-    Angle:         		195.2955780029297 kJ / mol
-    Torsion:       		13.520307540893555 kJ / mol
-    Nonbonded:     		None
-    vdW:           		43.82065216824412 kJ / mol
-    Electrostatics:		-707.1108989715576 kJ / mol
+    Bond:            17.31004524230957 kJ / mol
+    Angle:           195.2955780029297 kJ / mol
+    Torsion:         13.520307540893555 kJ / mol
+    Nonbonded:       None
+    vdW:             43.82065216824412 kJ / mol
+    Electrostatics:  -707.1108989715576 kJ / mol
```

Comment:
I think these may be exact output characters from a program. Is there a pressing need to modify this?

Reply:
Yes, this removes hard tabs.